doc/modules/ROOT/pages/tutorials/community-detection.adoc

```diff
@@ -58,7 +58,7 @@ gds = GraphDataScience(NEO4J_URI, auth=NEO4J_AUTH)
 ----
 from graphdatascience.server_version.server_version import ServerVersion
 
-assert gds.server_version() >= ServerVersion(1, 8, 0)
+assert gds.server_version() >= ServerVersion(2, 6, 0)
```

Comment:
```suggestion
assert gds.server_version() >= ServerVersion(2, 5, 0)
```
https://github.com/neo-technology/graph-analytics/blob/293a1ba53b88aa9e138f085ac32fbcd574668f7e/release/archive/2.5.0.md#L149

Reply:
ah these are the generated tutorials. will comment on the notebook.